src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -340,309 +334,235 @@
+                    //height: 100vh;
+                    border-right: $clr-default-borderwidth dotted $clr-action-blue-lighter;
                 }
+                .exceeded-max {
+                    border-right: $clr-default-borderwidth dotted rgba($clr-red-light-midtone, 0.3);
+                }
+            }
 
-                .action-item {
-                    @include clr-getTypePropertiesForDomElement(dropdown_text, (color, font-size, letter-spacing));
-
-                    background: transparent;
-                    border: 0;
-                    color: $clr-app-font-color-primary;
-                    cursor: pointer;
-                    display: block;
-                    line-height: calc(#{clr-getTypePropertyValueForDomElement(dropdown_text, line-height)} - 1px);
-                    margin: 0;
-                    padding: 1px 1rem 0;
-                    text-align: left;
-                    width: 100%;
-
-                    &:hover,
-                    &:focus {
-                        text-decoration: none;
-                        background-color: $clr-datagrid-action-popover-hover-color;
-                    }
-
-                    &.active {
-                        background: $clr-datagrid-row-hover;
-                        color: $clr-black;
-                    }
+            .datagrid-signpost-trigger .signpost {
+                margin: -0.3rem 0;
+                height: 1.03rem;
+                .signpost-trigger {
+                    height: inherit;
+                    line-height: 1rem;
+                }
+            }
+        }
 
-                    &:focus {
-                        outline: 0;
-                    }
+        .datagrid-cell {
+            flex: 1 1 auto;
+            text-align: left;
+            min-width: 4rem;
+            border-top: 0;
 
-                    &.disabled,
-                    &:disabled {
-                        cursor: not-allowed;
-                        opacity: 0.4;
-                        user-select: none;
+            &.datagrid-fixed-column {
+                flex: 0 0 $clr-datagrid-fixed-column-size;
+            }
 
-                        &:hover {
-                            background: none;
-                        }
+            &.datagrid-fixed-width {
+                flex: 0 0 auto;
+            }
 
-                        &:active,
-                        &:focus {
-                            background: none;
-                            box-shadow: none;
-                        }
-                    }
+            &--hidden {
+                display: none;
+            }
 
-                    clr-icon {
-                        vertical-align: middle;
-                        transform: translate3d(0px, -1px, 0);
-                    }
-                }
+            &.datagrid-fixed-column {
+                flex: 0 0 $clr-datagrid-fixed-column-size;
+                // IE10 needs this
```

Comment:
Do we need to support IE10 here?

Reply:
Will remove the comment. We need the min/max widths set.